app/plugin_install.go

```diff
@@ -138,5 +138,11 @@ func (a *App) removePlugin(id string) *model.AppError {
 		return model.NewAppError("removePlugin", "app.plugin.remove.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
+	// Remove bundle from the file store.
+	storePluginFileName := filepath.Join("./plugins", manifest.Id) + ".tar.gz"
+	if err := a.RemoveFile(storePluginFileName); err != nil {
```

Comment:
I believe this will error if the plugin doesn't exist, but I think we should treat that as a no-op, in particular to handle existing plugins that were never put into the filestore.

Reply:
I've added a guard condition to only delete if the bundle exists in the file store. This would allow to be back-ward compatible with existing plugins that have not been stored in the file store.